### PR TITLE
DM-802 - New docker registry

### PR DESCRIPTION
**Pull request by andresmoschini**
It is based on https://github.com/MakingSense/doppler-for-shopify/pull/341

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -28,38 +28,54 @@ pipeline {
...
+                        }
+                    }
+                }
+                stage('Publish pre-release images from INT') {
+                    when {
+                        branch 'INT'
+                    }
+                    steps {
+                        withDockerRegistry(credentialsId: "${DOCKER_CREDENTIALS_ID}", url: "") {
+                            sh 'sh build-n-publish.sh --image=${DOCKER_IMAGE_NAME} --image=${DOCKER_IMAGE_NAME} --commit=${GIT_COMMIT} --name=INT'
```
Comment (ms-darianbenito) [inline Jenkinsfile:63]: image param is duplicated

File: Jenkinsfile [hunk 2]
```diff
@@ -78,6 +78,56 @@ pipeline {
...
```
Comment (ms-darianbenito) [inline Jenkinsfile:113]: image param is duplicated

Comment (andresmoschini) [reply Jenkinsfile]: Good catch! fixed.

File: Jenkinsfile (hunk 1 above)
Comment (andresmoschini) [reply Jenkinsfile]: Good catch! fixed.

**Comment by ms-darianbenito** [comment]
✔️

**Comment by rockambole86** [comment]
LGTM 👍

**Comment by jhoffman-ms** [comment]
👍

**Pull request merged**